test/integration/api/forms/forms.js

```diff
@@ -438,6 +438,59 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
         });
     }));
 
+    it('should reject form with missing meta group', testService(async (service) => {
```

Comment:
can you please add another test case:
- publish a form with meta group
- upload another draft for the same form without meta group, it should be rejected